### DISCUSSION TITLE
stuck on UEFI boot screen

### DISCUSSION THREAD

**Discussion by budney1** [utmapp/UTM#3880] [UTM Help (macOS)]
Just got a new MacBook Pro M1 and I'm trying to run Windows 10 to use some legacy software I need.

I've never used UTM before, previously used VMWARE Fusion but that won't work anymore.

I managed to get Windows 10 installed and working but it was really rather laggy so some research let me to the arm64 version.
However something happened when I was uninstalling the previous VM of windows 10, and now whatever I do every VM gets stuck on the UEFI boot screen. If I disable UEFI boot then some VM's will load, but I didnt have to do that previously. The ARM64 iso I have of windows just won't get past the UEFI boot. I must have changed something but I've tried to uninstall all UTM files, and still it gets stuck every single time.

Not sure what I must have done wrong but is there a way to restore all settings to default?

Monterey 12.2.1 
UTM 3.1.5

thanks

**Comment by osy** [comment]
Can you make sure you don’t have “disable hypervisor” selected in the global settings (Cmd+,)?

**Reply by budney1** [reply]
on the preferences of the VM, use hypervisor is selected. Its odd that any VM I try and use now get stuck at the UEFI screen, when it never used to, something must have changed somewhere
